cmd/spire-agent/cli/command/run.go

```diff
@@ -43,6 +46,7 @@ type CmdConfig struct {
 	PluginDir  string
 	LogFile    string
 	LogLevel   string
+	Umask      string
```

Comment:
We can make this an `int` here and let the flags package handle typing for us so we don't have to attempt a conversion later on

Reply:
`string` is handy here because its default value (empty) can be used to indicate umask should not be changed. It's trickier with an `int` because its default value (0) is a valid umask value.